### DISCUSSION TITLE
CrashLoopBackOff Error in AKS

### DISCUSSION THREAD

**Discussion by experimentsbyfuture** [madhuakula/kubernetes-goat#67] [Q&A]
Hello, 
I am trying to deploy the Kubernetes Goat in my AKS cluster and keep getting errors for the three deployments below? I have tried this multiple times and I get the same CrashLoopBackOff error. Any insight here?
<img width="447" alt="goaterror" src="https://user-images.githubusercontent.com/110102885/187505225-0d98c422-eac5-44ee-a103-fe211e367c27.png">

**Comment by jorn-k** [comment]
Could this maybe be an issue with your runtime? (docker vs containerd, for example)

**Reply by experimentsbyfuture** [reply]
I think this is the issue. Is there a recommended approach to fix this? I am working with a colleague on this, and we ran into another error about failing to pull the image even after bypassing the reference to the docker socket.

**Reply by madhuakula** [reply]
Hey @futuretesting, Could you please share the events/logs from the deployment?
